frontend/src/exporter/Exporter.tsx

```diff
@@ -83,13 +84,16 @@ export function Exporter(props: ExportedData): JSX.Element {
                     placement={type === ExportType.Image ? DashboardPlacement.Export : DashboardPlacement.Public}
                 />
             ) : recording ? (
-                <SessionRecordingPlayer
-                    playerKey="exporter"
-                    sessionRecordingId={recording.id}
-                    mode={SessionRecordingPlayerMode.Sharing}
-                    autoPlay={false}
-                    noInspector={!showInspector}
-                />
+                <>
+                    <EmbeddedReplayHelper />
```

Comment:
as a sanity check would be nice to conditionally render this like... have a new mode for the exporter instead of sharing 

but that's nit picking, it doesn't seem expensive to render this